code/modules/antagonists/cult/cult.dm

```diff
@@ -367,18 +368,24 @@
 	explanation_text = "Summon Nar'Sie by invoking the rune 'Summon Nar'Sie'. <b>The summoning can only be accomplished in [english_list(summon_spots)] - where the veil is weak enough for the ritual to begin.</b>"
 
 /datum/objective/eldergod/check_completion()
+	if(killed)
+		return -1 // You failed so hard that even the code went backwards.
 	return summoned || completed
 
 /datum/team/cult/proc/check_cult_victory()
 	for(var/datum/objective/O in objectives)
+		if(O.check_completion() == -1)
+			return -1
 		if(!O.check_completion())
 			return FALSE
 	return TRUE
```

Comment:
need to undo the T/Fs here, it's no longer a boo

Reply:
not booleans (still)

Reply:
Odd, thought I had got that. It's resolved now.